lib/tail/index.ts

```diff
@@ -1,3 +1,7 @@
 import { AnyArray } from "../any-array";
```

Comment:
please add the changelog with the clear description

Reply:
Added the changeset in [this commit](https://github.com/ts-essentials/ts-essentials/pull/424/commits/b1213c5e6788571b98f3783d1d2390aca437a805), please verify once.

Reply:
Great work, LGTM!